srcpkgs/zathura-pdf-mupdf/template

```diff
@@ -1,19 +1,22 @@
 # Template file for 'zathura-pdf-mupdf'
 pkgname=zathura-pdf-mupdf
 version=0.3.4
-revision=1
+revision=2
 build_style=meson
 hostmakedepends="pkg-config"
-makedepends="mupdf-devel zathura-devel libressl-devel"
-depends="zathura"
+makedepends="mupdf-devel zathura-devel libressl-devel libopenjpeg2-devel
+ libjpeg-turbo-devel libjbig2dec"
+depends="zathura mupdf-devel"
```

Comment:
is it really required to depend on a -devel package ?

Reply:
Gonna test without, as I don't remember adding it as a dep

Reply:
Works fine without